Emel/index.html

```diff
@@ -0,0 +1,259 @@
+      <div class="col-sm">
+        <select class="form-select" id="accomodation" required>
+          <option value="accomodation_Yes">Yes</option>
+          <option value="accomodation_No">No</option>
+          </select>
+      </div>
+
+      <br>
+
+      <!--Diet-->
+      <div class="col-sm">
+        Dietary Preference:
+      </div>
+      <div class="col-sm">
+        <select class="form-select" id="diet" required>
+          <option value="Vegeterian">Vegeterian</option>
+          <option value="Non-Vegeterian">Non-Vegeterian</option>
+          </select>
+      </div>
+
+      <br>
+
+      <!--Category-->
+      <div class="col-sm">
+        Category of Registration*:
+      </div>
+      <div class="col-sm">
+        <select class="form-select" id="category_of_registration" required>
+          <option value="category_of_registration_General">General</option>
+          <option value="category_of_registration_ICC Member">ICC Member</option>
+          <option value="category_of_registration_Student">Student</option>
+          </select>
+      </div>
+
+      <br>
+
+      <!--Fees-->
+      <div class="col-sm">
+        Registration Fees:
+      </div>
+      <div class="col-sm">
+            <table class="table">
+              <thead>
+                <tr>
+                  <th scope="col">Category</th>
+                    <th scope="col">Before 31st Dec, 2021</th>
+                    <th scope="col">Before 27th Jan, 2022</th>
+                    <th scope="col">Spot Registration</th>
+                  </tr>
+                </thead>
+                <tbody>
+                <tr>
+                  <th scope="row">General</th>
+                    <td>INR 6000</td>
+                    <td>INR 7000</td>
+                    <td>INR 8000</td>
+                </tr>
+                <tr>
+                  <th scope="row">ICC Members</th>
+                    <td>INR 5000</td>
+                    <td>INR 6000</td>
+                    <td>INR 7000</td>
+                </tr>
+                <tr>
+                  <th scope="row">Students</th>
+                  <td>INR 3000</td>
+                  <td>INR 3500</td>
+                  <td>INR 3500</td>
+                </tr>
+                <tr>
+                  <th scope="row">Course Fee</th>
+                  <td>INR 1000</td>
+                  <td>INR 1000</td>
+                  <td>INR 1000</td>
+                </tr>
+                  </tbody>
+                </table>
+      </div>
+
+      <br>
+
+      <!--Mode of payment-->
+      <div class="col-sm">
+        Mode of Payment*:
+      </div>
+      <div class="col-sm">
+        <select class="form-select" id="mode_of_payment" required>
+          <option value="mode_of_payment_NEFT">NEFT</option>
+          <option value="mode_of_payment_RTGS">RTGS</option>
+          <option value="mode_of_payment_IMPS">IMPS</option>
+          <option value="mode_of_payment_Demand Draft">Demand Draft</option>
```

Comment:
there should not be any whitespaces in the value

Reply:
White spaces? I have connected everything with underscore right?
Should I remove that and just keep it together?

Reply:
there is a whitespace in line 226 between Demand and Draft

Reply:
Ok, I will do that.
One more doubt I had is:
Don't we have to validate the input?
Won't this form get submitted even if I Ieave a mandatory field blank?

Reply:
Mandatory fields are given the argument "required" (Eg. Line 57 or line 60 of your original commit). If you leave any "required" field blank and click the submit button, you will be shown a message "This field is required". You cannot bypass any "required" fields and submit.

The code you sent is calling a js file for form-validation in line 257. This file however is not included in your commit. Hence validation will not work for the commit. You can either remove line 257 or include the file along with your commit. My suggestion is to remove the line as "required" is already doing the job.